spec/classes/server_spec.rb

```diff
@@ -31,7 +31,7 @@
             :servertype   => server_type,
             :storeconfigs => 'puppetdb',
             :environmentpath => '$confdir/environments',
-            :basemodulepath => '/etc/puppet/modules:/usr/share/puppet/modules',
+            :basemodulepath => ['/etc/puppet/modules', '/usr/share/puppet/modules'],
```

Comment:
you could even add a second test, since both, string and array will work this way